cloudsmith/resource_repository_upstream.go

```diff
@@ -14,19 +14,23 @@ import (
 
 // upstream types
 const (
```

Comment:
Should we also include `hex` in this list?

It's already exposed via via the CLI to create upstream resources
https://github.com/cloudsmith-io/cloudsmith-cli/blob/76c4dfe31de68753ea30398c2f5973ece75d05b9/cloudsmith_cli/cli/commands/upstream.py#L28

Reply:
Totally, thanks for bringing it up Kyle!